lib/kontena/machine/azure/cloudinit.yml

```diff
@@ -37,14 +37,21 @@ coreos:
         Restart=always
         RestartSec=5
         ExecStart=/usr/bin/docker logs --tail=10 -f kontena-etcd
-    - name: 10-weave.network
+    - name: 50-docker.network
+      mask: true
+    - name: 50-docker-veth.network
+      mask: true
+    - name: zz-default.network
       runtime: false
       content: |
+        # default should only match real network interfaces
         [Match]
-        Type=bridge
-        Name=weave*
-
+        Name=eth*
         [Network]
+        DHCP=yes
```

Comment:
I assume there's some coreos oem-cloudconfig that configures the `eth` interfaces, and not DHCP?